generator/main.py

```diff
@@ -210,12 +210,12 @@ def preprocess(self, source):
         for line in lines:
             # The capture in the regex adds one element per block start or end so we divide by two
             # there is also an extra line chunk corresponding to the line end, so we substract it.
-            numends = (len(self.blockend.split(line)) - 1) / 2
+            numends = int((len(self.blockend.split(line)) - 1) / 2)
```

Comment:
IIRC this can be done with `(len(self.blockend.split(line)) - 1) // 2` in both python 2 and 3. Do whichever you think is clearer, I don't have a strong feeling.